src/styles/style.css

```diff
@@ -0,0 +1,365 @@
+*,
+*::after,
+*::before {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+.nav {
+  display: flex;
+  position: fixed;
+  height: 64px;
+  width: 100%;
+  background-color: var(--blue-100);
+  box-shadow: 2px 2px 2px #4a54b1;
+  justify-content: space-between;
+  transition: 0.5s;
+  z-index: 1;
+}
+.logo {
+  padding-left: 18px;
+  padding-top: 22px;
+}
+.nav ul {
+  display: flex;
+  padding-right: 18px;
+  list-style-type: none;
+}
+.nav ul li {
+  margin: 0px 15px;
+  line-height: 64px;
+  color: var(--blue-600);
+  text-decoration: none;
+  font-family: "Lato", sans-serif;
+}
+li:hover {
+  border-bottom: 4px solid var(--blue-500);
+  cursor: pointer;
+}
+.main {
+  padding: 23px 26px 0 26px;
+  display: flex;
+  flex-direction: column;
+}
+.square_primary {
+  aspect-ratio: 1.7/1;
+  background-color: var(--gray-200);
+  border-radius: 8px;
+  margin-bottom: 16px;
+  margin-top: 80px;
+}
+.title {
+  font-size: 20px;
+  padding-bottom: 14px;
+  font-family: "Montserrat", sans-serif;
+}
+.title::after {
+  padding-top: 6px;
+  display: block;
+  padding-bottom: 2px;
+  content: "";
+  position: relative;
+  width: 128px;
+  border-bottom: 2px solid var(--orange-500);
```

Comment:
Although it's not required, you should always try to place CSS properties in some order, e.g.  things related to display, height & width goes first, then position stuff & fonts later.